msa-persistence/src/main/java/cz/muni/fi/pa165/monsterslayeragency/dao/JobDao.java

```diff
@@ -0,0 +1,7 @@
+package cz.muni.fi.pa165.monsterslayeragency.dao;
+
+/**
+ * @author Michaela Bajanova (469166)
+ */
+public class JobDao {
```

Comment:
Class should be named like <entity name><Repository> e.g. `JobRepository` so that we maintain the naming convention in our classes, as well as because DAO and Repository is not the same thing in Hibernate.